vue-training/src/scss/equalizer.scss

```diff
@@ -0,0 +1,92 @@
+$equalizer-black: #000;
```

Comment:
you can use [wolox-equalizer](https://www.npmjs.com/package/wolox-equalizer) package here